docs/dev/api/rdc.md

```diff
@@ -597,3 +597,113 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
 </details>
 
 ---
+
+### Stop a Job
+
+<details><summary><span className="api put">PUT</span> <code>/v1/rdc/jobs/&#123;job_id&#125;/stop</code></summary>
+<p/>
+
+Stops a running job described by the `job_id`.
+
+**_NOTE:_** Right now only Appium jobs can be stopped while they are actively running.
+
+#### Parameters
+
+<table id="table-api">
+  <tbody>
+    <tr>
+     <td><code>job_id</code></td>
+     <td><p><small>| PATH | REQUIRED | STRING |</small></p><p>The unique identifier of a job running on a real device in the data center. You can look up job IDs using the <a href="#get-real-device-jobs">Get Real Device Jobs</a> endpoint.</p></td>
+    </tr>
+  </tbody>
+</table>
+
+<Tabs
+groupId="dc-url"
+defaultValue="us"
+values={[
+{label: 'United States', value: 'us'},
+{label: 'Europe', value: 'eu'},
+]}>
+
+<TabItem value="us">
+
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request PUT 'https://api.us-west-1.saucelabs.com/v1/rdc/jobs/293d84fb2f634ff29a750c3f8eaee592/stop' | json_pp
+```
+
+</TabItem>
+
+<TabItem value="eu">
+
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request PUT 'https://api.eu-central-1.saucelabs.com/v1/rdc/jobs/48c6d12f9ef944439453b5abc6715b54/stop' | json_pp
+```
+
+</TabItem>
+</Tabs>
+
+#### Responses
+
+<table id="table-api">
+<tbody>
+  <tr>
```

Comment:
@bahrimootaz 

I'm getting different responses than we have in the docs below

```bash
~/Sauce/vusb ❯ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \                                                                                                
--request PUT 'https://api.eu-central-1.saucelabs.com/v1/rdc/jobs/9bcd27aca3354b16b80eee742ac230da/stop' | json_pp
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
malformed JSON string, neither array, object, number, string or atom, at character offset 0 (before "(end of string)") at /usr/bin/json_pp5.30 line 52.
~/Sauce/vusb ❯ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \                                                                                                
--request PUT 'https://api.eu-central-1.saucelabs.com/v1/rdc/jobs/9bcd27aca3354b16b80eee742ac230da/stop' | json_pp
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   148    0   148    0     0    768      0 --:--:-- --:--:-- --:--:--   787
{
   "code" : "BAD_REQUEST",
   "message" : "The test cannot be stopped because the test execution has already finished. Only tests in progress can be stopped"
}
```